docs: updated android-core docs (#289)
dyte-io/docs: docs/android-core/quickstart.mdx

The `DyteMobileClient` is the main class of the SDK. It is the entry point and
the only class required to initialize Dyte SDK.

<Tabs groupId="build-android-core">
  <TabItem value="kotlin" label="Kotlin" default>

```kotlin
val dyteClient = DyteMeetingBuilder.build(activity)
```

  </TabItem>
  <TabItem value="java" label="Java">

```java
DyteMobileClient dyteClient = DyteMeetingBuilder.build(activity);
```

  </TabItem>
</Tabs>

## Step 3: Configure a Dyte meeting

Set the properties in the `DyteMeetingInfoV2` class. You just need to provide the
participant's `authToken`.

| Name        | Description                                                                                                                                                                                                                                                                            |
| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |

> | `authToken` | After you've created the meeting, add each participant to the meeting using the [Add Participant API](/api?v=v2#/operations/add_participant) The API response contains the `authToken`. |     |

Reviewer: ```suggestion
| `authToken` | After you've created the meeting, add each participant to the meeting using the [Add Participant API](/api?v=v2#/operations/add_participant). The API response contains the `authToken`. |     |
```

| `enableAudio` | You can control the state of audio track by passing `true` or `false` here. |     |

> | `enableVideo` | You can control the state of video track by passing `true` or `false` here. |     |

Reviewer: ```suggestion
| `enableVideo` | Set whether to join the meeting with your Camera ON or OFF by passing `true` or `false`. |     |
```

OR

```suggestion
| `enableVideo` | Set initial state of your Mic to join the meeting with, by passing `true` or `false`. |     |
```

Could come up with only these to describe the flag.

| `baseUrl` | Base URL of the dyte's enviorment you have created the meeting on. |     |

<Tabs groupId="meeting-props-android-core">
  <TabItem value="kotlin" label="Kotlin" default>

```kotlin
val meetingInfo =
      DyteMeetingInfoV2(
        authToken = AUTH_TOKEN, 
        enableAudio = true, 
        enableVideo = true, 
        baseUrl = "dyte.io"
      )
```

  </TabItem>
  <TabItem value="java" label="Java">

```java
DyteMeetingInfoV2 meetingInfo = new DyteMeetingInfoV2(
          MeetingConfig.AUTH_TOKEN, // auth_token
          true,   // enableAudio
          true,   // enableVideo
          "dyte.io" // baseUrl
          );
```

  </TabItem>
</Tabs>

## Step 4: Initialize the Dyte meeting

> To initialize the meeting, call the `init()` method obtained on
> `dyteClient` with the `meetingInfo` argument. This will establish the connection
> with the Dyte meeting server.

Reviewer: ```suggestion
To initialize the meeting, call the `init()` method on the `dyteClient` object with the `meetingInfo` argument. This establishes a connection with the Dyte meeting server.
```
